src/sidebar/components/test/annotation-user-test.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const { createElement } = require('preact');
+const { shallow } = require('enzyme');
+
+const AnnotationUser = require('../annotation-user');
+
+describe('AnnotationUser', () => {
+  let fakeAnnotation;
+  let fakeFeatures;
+  let fakeIsThirdPartyUser;
+  let fakeServiceUrl;
+  let fakeSettings;
+  let fakeUsername;
+
+  const createAnnotationUser = () => {
+    return shallow(
+      <AnnotationUser
+        annotation={fakeAnnotation}
+        features={fakeFeatures}
+        serviceUrl={fakeServiceUrl}
+        settings={fakeSettings}
+      />
+    ).dive(); // Dive needed because of `withServices` usage in component
+  };
+
+  beforeEach(() => {
+    fakeAnnotation = {
+      user: 'someone@hypothes.is',
+    };
+    fakeFeatures = { flagEnabled: sinon.stub() };
+    fakeIsThirdPartyUser = sinon.stub();
```

Comment:
I would suggest configuring `fakeIsThirdPartyUser` to return an explicit value so that it is obvious whether tests get a first or third-party user by default.